teamPage.css

```diff
@@ -1,221 +1,367 @@
 @import url('https://fonts.googleapis.com/css2?family=Heebo:wght@300;400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display&display=swap');
 
-:root  {
-  --dark:#21243D;
-  --coral:#Ff6464;
+:root {
+  --dark: #21243d;
+  --coral: #ff6464;
+  --red: #f8593b;
   --white: #f5f5f5;
---purple:#434ba1;
+  --purple: #434ba1;
+  --yellow: #ffc22b;
+  --pure-white: #ffff;
+  --grey: rgba(0, 0, 0, 0.6);
+  --light-grey: rgba(0, 0, 0, 0.2);
 }
 
 /* note we are using em's as our units of measurement in this project( 16px = 1.00em ).*/
 
- /* We are also using a mobile first approach where styles outside of a media queries are targeted at the smallest screen size and progressively laregr screen sizes are inside of the appropriate media queries  */
+/* We are also using a mobile first approach where styles outside of a media queries are targeted at the smallest screen size and progressively laregr screen sizes are inside of the appropriate media queries  */
```

Comment:
This is not correct, we're not using a mobile first approach. We're training on HTML/CSS and the project is designed to match the skills of our group members. The comment on line 17 is helpful for converting em's and px's but line 19 needs to be removed, please